sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/ConfigurableSpanExporterProvider.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.autoconfigure.spi;
+
+import io.opentelemetry.sdk.autoconfigure.ConfigProperties;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+/**
+ * A service provider interface (SPI) for providing additional exporters that can be used with the
+ * autoconfigured SDK. If the {@code otel.trace.exporter} property contains a value equal to what is
+ * returned by {@link #getName()}, the exporter returned by {@link
+ * #createExporter(ConfigProperties)} will be enabled and added to the SDK.
+ */
+public interface ConfigurableSpanExporterProvider {
```

Comment:
I know there's precedence with `ConfigurablePropagatorProvider`, but I think the `Configurable` prefix is kinda redundant/unnecessary.  We're already in the autoconfigure package space.  What about simply `SpanExporterProvider`?

Reply:
@anuraaga your opinion?

Reply:
Without configurable, I'd expect the exporter to be automatically enabled, not only made configurable. Not a huge deal either way though.

Reply:
I'm not sure why the configurability speaks to the idea of enablement or not....but I am fine leaving it for now since there is prior art.  If it bothers us later, we can always change it.